default.css

```diff
@@ -80,6 +83,26 @@ html {
   background-color: var(--success) !important;
 }
 
+.bg-purple{
+  background-color: var(--purple) !important;
```

Comment:
The use of  `! important`  is not a great practice I guess. I don't know your use case but maybe try to find a way around it.